docs/cli_zh.md

```diff
@@ -1,3 +1,79 @@
 # Fling 的 CLI 使用
 
-TBD
+在这份文档中，我们将介绍 Fling 中命令行界面（CLI）的使用方式。
+
+## 版本
```

Comment:
Explain this clearer, such as "Check the version of Fling", also for English doc.